doc/services/pm/device_runtime.rst

```diff
@@ -26,6 +26,10 @@ asynchronously, it will be put into the
 :c:enumerator:`PM_DEVICE_STATE_SUSPENDING` state first and then into the
 :c:enumerator:`PM_DEVICE_STATE_SUSPENDED` state when the action is run.
 
+If the device is connected to a power domain and device runtime power management
```

Comment:
It's not wrong but it could be more complete:
```
For devices on a power domain (via the devicetree 'power-domain' property), device runtime
power management automatically attempts to request and release the dependent domain
in response to :c:function:`pm_device_runtime_get` and :c:function:`pm_device_runtime_put`
calls on the child device.
```

Reply:
Thats a great suggestion. For me it was important to also mention that the domain itself needs the runtime management enabled. This is missing in your suggestion. I needed quite a while of debuging till I understood this.

Reply:
Appending the following is probably sufficient:
```
For the previous to automatically control the power domain state, device runtime PM must be enabled
on the power domain device (either through the `zephyr,pm-device-runtime-auto` devicetree property
or :c:function:`pm_device_runtime_enable`)
```